hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtSoftwareModuleMapper.java

```diff
@@ -156,28 +155,20 @@ public static MgmtSoftwareModule toResponse(final SoftwareModule baseSofwareModu
      * @return
      */
     static MgmtArtifact toResponse(final Artifact artifact) {
-        final MgmtArtifact.ArtifactType type = artifact instanceof LocalArtifact ? MgmtArtifact.ArtifactType.LOCAL
-                : MgmtArtifact.ArtifactType.EXTERNAL;
-
         final MgmtArtifact artifactRest = new MgmtArtifact();
-        artifactRest.setType(type);
```

Comment:
Well, it was an unused part of the API. So I think its save to remove it. We supported only one type (local) anyway.